### PR TITLE
Added integration tests

### PR DESCRIPTION
**Pull request by jhellar**
## Motivation
Enable integration tests also for sdk.

### CODE REVIEW

**Comment by wtrocki** [comment]
@jhellar Somehow jenkins did not pick up the label :((

**Comment by jhellar** [comment]
@wtrocki I asked Pavel - Jenkins build is not re-triggered when label is added, it has to be done manually. Also I just realized that this can not be properly tested until https://github.com/aerogear/cordova-showcase-template/pull/25 is merged.

**Comment by wtrocki** [comment]
@jhellar it is possible by doing integration in github and detect labels. See: https://github.com/aerogear/aerogear-js-sdk/settings/installations

We currently using integration from Jenkins side.

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    stage ('Install dependencies') {
+      sh '''
+        npm install
+        npm run bootstrap
+        npm run build
+      '''
+      dir('cordova-example') {
+        sh '''
+          npm install
+          npm link ../packages/core
```
Comment (josemigallas) [inline Jenkinsfile]: We should about this IMO, we should test the available packages in NPM. We could do a pre-release before doing integration, then the full release.

Comment (wtrocki) [reply Jenkinsfile]: @josemigallas Absolutely not. This is for testing PR's with changes that are not released. If we going to change purpose of this test then we can change that.

Comment (josemigallas) [reply Jenkinsfile]: OK

**Comment by wtrocki** [comment]
@jhellar We did not get any response for this PR for 20 days. Closing as outdated!
Feel free to reopen.

**Pull request closed without merging**